tests/terraformtests/bin/run_go_test

```diff
@@ -19,5 +19,9 @@ PATCH="etc/0001-Patch-Hardcode-endpoints-to-local-server.patch"
 (
 cd terraform-provider-aws || exit
 echo "Running tests $2 for service $1..."
-AWS_ACCESS_KEY_ID=test AWS_SECRET_ACCESS_KEY=test TF_ACC=true go test ./internal/service/$1/ -v -timeout 60m -run $2
+TF_ACC=1 \
+  AWS_ACCESS_KEY_ID=test AWS_SECRET_ACCESS_KEY=test AWS_DEFAULT_REGION=us-west-1 \
```

Comment:
I think this is why some of the other TF tests are now failing - not all services are available in `us-west-1`. Can we keep `us-east-1` as the default?

Reply:
![image](https://github.com/getmoto/moto/assets/13847870/92ea35f5-e510-470f-a5be-d506cf96d51a)

- Region and Alternate Region should be different.

Reply:
@bblommers , Only one job is failing. Can't find traces of the variable issue in here. Can you please take a look?

Reply:
Those two tests (`TestAccRoute53ZoneAssociation_crossAccount` and `TestAccRoute53ZoneAssociation_crossAccountAndRegion`) used to be skipped, because `AWS_ALTERNATE_ACCESS_KEY_ID` was not set. 

Now that the variable is set, they are executed - and fail because Moto doesn't yet support all required methods.

I'll just merge this now, and remove these tests from the list afterwards.

Thanks for getting this working @macnev2013!